### ISSUE TITLE
Parking Lot

### ISSUE THREAD

**Issue by ericalloyd** [githubschool/mayhem#7]
## Post any questions you may have here during the training.

**Comment by roywu1999** [comment]
Could we cover the topic for CI/CD?

**Comment by dumitrescucristina** [comment]
Why readme file is read only when we clone?

**Comment by ericalloyd** [comment]
Topics of Interest for Day 2:
@alireamandegar `git stash`
@Alquh `git rebase`
Victor R: **GitHub Actions** (Adding JIRA ticket # to commit message automatically)

Open Questions:
@liaolinda Is there an option for `git remote set-url` to set the `fetch` and `push` repos separately?

**Comment by ericalloyd** [comment]
> Open Questions:
> @liaolinda Is there an option for `git remote set-url` to set the `fetch` and `push` repos separately?

Yes! With the `--push` option you can specify that a repo url is for push actions only. 

Let's say you have one repo `/foobar1` set up for both push and fetch:
```
> git remote -v
origin  https://github.com/example/foobar1.git (fetch)
origin  https://github.com/example/foobar1.git (push)
```

Here's the command to **change** the push repo url to `/foobar2`:
`git remote set-url --push origin https://github.com/example/foobar2.git`

Your output will now look like this:
```
> git remote -v
origin  https://github.com/example/foobar1.git (fetch)
origin  https://github.com/example/foobar2.git (push)
```


To add **additional** repos (vs. overwriting the existing ones), use `--add`:

`git remote set-url --add --push origin https://github.com/example/foobar3.git`

Your output should then look like this:
```
> git remote -v 
origin  https://github.com/example/foobar1.git (fetch)
origin  https://github.com/example/foobar2.git (push)
origin  https://github.com/example/foobar3.git (push)
```
Now, each time you `git push`, your branch will be pushed to _both_ the `/foobar2` and `/foobar3` repos.

Thanks for this great question!

**Comment by ericalloyd** [comment]
### Day 1 Recording
**Link**: https://github.zoom.us/rec/share/puIuNu2q_FJLRtbzynHNRLYhH62-aaa82nMerPcLmkpPbcxqP-eO-ghwUlOyrkqU 
**Access Password**: U6!8&971

**Comment by ericalloyd** [comment]
@roywu1999 How long will we have access to these repos?

I'm not aware of any expiration but will verify this!
